src/main/webapp/app/entities/quiz-exercise/quiz-exercise.module.ts

```diff
@@ -44,6 +45,7 @@ const ENTITY_STATES = [...quizExerciseRoute, ...quizExercisePopupRoute];
         QuizExerciseResetDialogComponent,
         QuizExerciseResetPopupComponent,
         QuizExerciseDetailComponent,
+        QuizExerciseExportComponent,
```

Comment:
I am not entirely sure why I had to do this, but without it I got the error message `Component QuizExerciseExportComponent is not part of any NgModule or the module has not been imported into your module.` when running the tests in `tutor-exercise-dashboard.spec.t`. However, the tutor exercise dashboard should not be related to the quiz stuff in any way.

Reply:
You are right, I think this was also an issue with the entry components.